src/style.css

```diff
@@ -1 +1,114 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+:root {
+  --main-color: #3d4e61;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  color: var(--main-color);
+  font-weight: 300;
+}
+
+.search {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto;
+}
+
+.search--big {
+  height: 70px;
+  width: 100%;
+}
+
+.search--small {
+  height: 42px;
+  width: 100%;
+}
+
+.search__label {
+  position: relative;
+  width: 100%;
+  display: block;
+}
+
+.search__label::after {
+  content: '';
+  position: absolute;
+  display: block;
+
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+}
+
+.search__label--big::after {
+  top: 25px;
+  left: 26px;
+  width: 19px;
+  height: 19px;
+}
+
+.search__label--small::after {
+  top: 15px;
+  left: 13px;
+  width: 11px;
+  height: 11px;
+}
+
+.search__input {
+  width: 100%;
+  height: 100%;
+  border: 1px solid #e1e7ed;
+  font-family: inherit;
+  font-weight: 300;
+  color: var(--main-color);
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  box-sizing: border-box;
+  border-radius: 4px;
+  outline: none;
+}
```

Comment:
According to the checklist, you should not fix the container size if there is no such a requirement. In your case, you are setting a fixed width and height to the `.search__input` class. This could potentially lead to overflow or accidental scroll bar.